src/data/git-quiz.ts

```diff
@@ -542,13 +542,13 @@ const gitQuiz = [
     Distractor2: "Nano",
     Distractor3: "VS Code",
     Explanation:
-      "Vim stands for Vi Improved and it is terminal based text editor used among Linux users. You can open the editor by using the following command: vi fileName",
+      "Vim stands for Vi Improved and is a terminal-based text editor used among Linux users. You can open the editor by using the following command: vi fileName",
     Link: "https://www.freecodecamp.org/news/learn-vim-beginners-tutorial/"
   },
   {
     Question: "How do you unstage a staged file in git?",
     Answer: "git restore --staged <file-path>",
-    Distractor1: "git unstage <file-path>",
+    Distractor1: "git unstaged <file-path>",
```

Comment:
this change is extraneous for this PR, please remove it

Reply:
Removed